qmxgraph/api.py

```diff
@@ -636,27 +639,27 @@ def on_terminal_with_port_changed(self, handler):
             id of the old terminal.
         """
         return self.call_api(
-            'onTerminalWithPortChanged', qmxgraph.js.Variable(handler))
+            'registerTerminalWithPortChanged', qmxgraph.js.Variable(handler))
```

Comment:
Just for consistency with others, `registerTerminalWithPortChangedHandler`

Reply:
Done!